plugins/reporters/spdx/src/funTest/kotlin/SpdxDocumentReporterFunTest.kt

```diff
@@ -70,9 +70,9 @@ class SpdxDocumentReporterFunTest : WordSpec({
                 fileInformationEnabled = false
             )
 
-            val document = fromJson<SpdxDocument>(jsonSpdxDocument)
+            val jsonDocument = fromJson<SpdxDocument>(jsonSpdxDocument)
 
-            document.files should beEmpty()
+            jsonDocument.files should beEmpty()
```

Comment:
Actually, shouldn't this be `spdxDocument` (if renaming it at all)? Because the data has already been deserialized at this point, so strict you cannot tell anymore that it is / was JSON.